Mods/SML/Source/SML/Private/Registry/ModContentRegistry.cpp

```diff
@@ -860,13 +860,39 @@ void UModContentRegistry::AutoRegisterRecipeReferences( TSubclassOf<UFGRecipe> R
 	}
 }
 
+static bool IsResourceFormFilteredOut(EResourceForm ResourceForm, EGetObtainableItemDescriptorsFlags Flags)
+{
+	using enum EGetObtainableItemDescriptorsFlags;
```

Comment:
This `using` directive is needed to avoid repeating `EGetObtainableItemDescriptorsFlags` when ORing various flags together. I'm not sure if `using` directives are against style guidelines (which?), but I think it's worth it here for the sake of brevity.